orders/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+from core.models import Base
+
+class Order(Base):
+    order_number = models.IntegerField()
+
+    class Meta:
```

Comment:
어떤 유저의 주문인지 알 수 있어야 할 것 같아요 희택님!
```suggestion
class Order(Base):
    order_number = models.IntegerField()
    user         = models.ForeignKey('users.User', ```)

    class Meta:
```

Reply:
수정하겠습니다!